hboie/jasper-report-bundle/1.1/config/packages/jasper_report.yaml

```diff
@@ -0,0 +1,5 @@
+hboie_jasper_report:
+    host:      'http://localhost:8080/jasperserver'
+    username:  jasperadmin
+    password:  jasperadmin
```

Comment:
Maybe add environment variables for host, username, password and org_id?

Does that make sense?